packages/client/src/components/app/Layout.svelte

```diff
@@ -281,7 +281,14 @@
         </div>
       </div>
     {/if}
-    <div class="main-wrapper">
+    <div
+      class="main-wrapper"
+      on:click={() => {
+        if ($builderStore.inBuilder) {
+          builderStore.actions.selectComponent(null)
```

Comment:
Could we send up `$builderStore.screen._id` here instead? Sending up `null` causes the builder to do an immediate redirect to the screen ID anyway, but we have the screen ID here so we can save the builder the extra work (and slight delay).